site/cypress/e2e/Cypress-Gradeable/hotkey.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('TA grading hotkey testing', () => {
+    after(() => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update?nav_tab=2']);
+        cy.get('[data-testid="peer-component-container"]').should('exist');
+        cy.get('[data-testid="peer-component-container"]').find('[data-testid="delete-gradable-component"]').click();
+        cy.logout();
+    });
+
+    it('toggle keyboard shortcut', () => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get('[data-testid="grade-button"]').eq(12).click();
+        cy.get('[data-testid="grading-panel-header"]').as('navigationBar');
+        cy.get('@navigationBar').type('{A}');
+        cy.get('[data-testid="autograding-results"]').should('contain', 'Autograding Testcases');
+        cy.get('@navigationBar').type('{G}');
+        cy.get('[data-testid="grading-rubric"]').should('contain', 'Grading Rubric');
+        cy.get('#edit-mode-enabled').should('not.be.checked');
+        cy.get('@navigationBar').type('{E}');
+        cy.get('#edit-mode-enabled').should('be.checked');
+        cy.get('@navigationBar').type('{O}');
+        cy.get('[data-testid="submission-browser"]').should('contain', 'Submissions and Results Browser');
+        cy.get('@navigationBar').type('{S}');
+        cy.get('[data-testid="student-info"]').should('contain', 'Student Information');
+        cy.get('@navigationBar').type('{X}');
+        cy.get('[data-testid="grade-inquiry-inner-info"]').should('contain', 'Grade Inquiry');
+        cy.get('@navigationBar').type('{T}');
+        cy.get('[data-testid="solution-ta-notes"]').should('contain', 'Solution/TA Notes');
+    });
+    it('testing discussion, peer and notebook', () => {
+        cy.login();
+        cy.visit(['sample', 'config']);
+        cy.get('[data-testid="forum-enabled"]').should('be.checked');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update']);
+        // adding the peer grading panel, discussion panel, notebook panel
+        cy.get('[data-testid="yes-discussion"]').check();
+        cy.get('[data-testid="yes-discussion"]').should('be.checked');
+        cy.get('[data-testid="page-2-nav"]').click();
+        cy.get('[data-testid="add-new-peer-component"]').click();
+        // cy.get('[data-testid="page-1-nav"]').click();
+        // cy.get('[data-testid="start-new-notebook"]').click();
+        // cy.get('[data-testid="markdown"]').click();
+        // cy.get('[data-testid="notebook-builder-markdown-0"]').click().type('Notebook-Cypress-Test');
+        // cy.get('[data-testid="notebook-save"]').click();
```

Comment:
Is this still being used? if so, uncomment it so we can see if the tests pass, if not, remove it.